Cargo.toml

```diff
@@ -21,3 +21,4 @@ openssl = { version = "0.7", optional = true }
 rustc-serialize = "0.3"
 url = "0.2"
 chrono = "0.2.15"
+log = "*"
```

Comment:
Can you specify `log = 0.3` here? One can't publish to crates.io with wildcard dependencies.